.vscode/tasks.json

```diff
@@ -0,0 +1,24 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build",
+            "type": "shell",
+            "command": "./build.sh -d",
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            },
+            "problemMatcher": [
+                "$gcc"
+            ]
+        },
```

Comment:
Is it possible to add a run task? Maybe another testing script to standup a udp server to interact with this lib if it could be integration tested independently?

Reply:
There is no executable. Just a library